instances/atlas/brainAtlasVersion/WHSSDatlas_v2.jsonld

```diff
@@ -266,20 +266,23 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite these publications: Papp et al., 2014 and Kjonigsen et al., 2015.",

```

Comment:
it is better to state the full citation. you don't want people to search for the papers (at least state also the DOI)

Reply:
Full citation has been added.